app/ui-react/syndesis/src/modules/integrations/components/editor/interfaces.ts

```diff
@@ -29,7 +29,9 @@ export enum DataShapeDirection {
 
 export interface IUIStep extends StepKind {
   board?: ConnectionBulletinBoard;
+  configRequired?: boolean;
   connector?: Connector;
+  isTechPreview?: boolean;
```

Comment:
I'd make this required, just to stay on the safe side and highlight any point where we missed passing the updated structure.